lib/plugins/pocketmine.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Scales.js — the flexible game management daemon built for PufferPanel.
+ * Licensed under a GPL-v3 license.
+ *
+ * pocketmine.js — Adds Pocketmine-MP server support to Scales.js
+ */
+var fs = require("fs-extra"),
+	path = require('path'),
+	log = require("../logger.js"),
+	properties = require("properties"),
+	gamedig = require("gamedig"),
+	settings = {};
+
+settings = {
+	name: "Pocketmine-MP",
+	stop: "stop",
+	exe: "java",
+	jar: "data/plugins/pocketmine/QuietPufferMine.jar",
+	cfg: "server.properties",
+	trigger: {
+		started: ')! For help, type "help" or "?"',
+		// eula: 'Go to eula.txt for more info.'
```

Comment:
@DaneEveritt For what it's worth: trailing commas work fine in JavaScript like that. 
```javascript
root@vinteum:~# node
> t = {a: 1, b:2, }
{ a: 1, b: 2 }
>
```